graphx/src/main/scala/org/apache/spark/graphx/lib/PageRank.scala

```diff
@@ -19,8 +19,11 @@ package org.apache.spark.graphx.lib
 
 import scala.reflect.ClassTag
 
-import org.apache.spark.graphx._
+import breeze.linalg.{Vector => BV}
+
+import org.apache.spark.ml.linalg.{Vector, Vectors}
```

Comment:
move `org.apache.spark.ml.linalg.{Vector, Vectors}` down, and move `org.apache.spark.graphx._` up. We have an automated system detecting if the import is alphabetical, and that is failing the build.